package.json

```diff
@@ -2,8 +2,9 @@
   "name": "octane-node",
   "version": "0.5.0",
   "description": "Node bindings for the Octane API",
-  "author": "support@getoctane.io",
-  "main": "index.js",
+  "author": "team@getoctane.io",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
```

Comment:
this might be the main line that fixes things, though i tweaked a few other things in this file and codegen caused some other changes to be propagated.